app/components/Form/Field.js

```diff
@@ -76,7 +76,10 @@ export type FieldProps = {
  *
  * http://redux-form.com/6.0.5/docs/api/Field.md/
  */
-export function createField(Component: ComponentType<*>) {
+export function createField(
+  Component: ComponentType<*>,
+  useLabel?: boolean = true
```

Comment:
I prefer an object over a simple boolean. Also, it would be nice with some docs about what `useLabel` does.

Reply:
agreed!